topology/Brandeis University/Brandeis Tier 3/Brandeis-Atlas-T3.yaml

```
@@ -1,9 +1,9 @@
 GroupDescription: Brandeis ATLAS Tier 3
 GroupID: 284
-Production: true
+Production: false
```

"Production" actually means "is it in the production pool (as opposed to the ITB pool)"
```suggestion
Production: true
```